syntax = "proto3";

package bookstore;

import "shared/order.proto";

service ExecutorService {
    
}